environments/icds-cas/app-processes.yml

```diff
@@ -377,3 +382,5 @@ pillows:
   'pillow15':
     FormSubmissionMetadataTrackerPillow:
       num_processes: 16
+  'pillow21':
+  'pillow22':
```

Comment:
These aren't in the inventory file. Also I don't have anything to add to them at the moment. I think we should be fine with our current setup until Sravan's work to combine processors is out in January